src/js/components/Grid/doc.js

```diff
@@ -64,8 +64,11 @@ space in the column axis.`,
     columns: PropTypes.oneOfType([
       PropTypes.arrayOf(
         PropTypes.oneOfType([
+          PropTypes.arrayOf(PropTypes.OneOfType([
```

Comment:
This seems unrelated with your PR. can you explain what you are doing here? The ts definitions don't need an update?

Reply:
The `ts` definition is correct :) because it is easier to write more general type requirements than this home-brew react-docgen replacement :)

In other words, this definition is not consistent with the ts definition in its original form.